### PR TITLE
Added analytics to the bosh release and automated all configurations including databases.

### PR DESCRIPTION
**Pull request by bhathiya**

### CODE REVIEW

File: bosh-release/README.md [hunk 1]
```diff
@@ -4,7 +4,18 @@ A BOSH release for deploying WSO2 API Manager 2.1.0 on BOSH Director:
...
+
+  a) Install BOSH CLI v2.x from [bosh.io](http://bosh.io/)
+  
+  b) Install Ruby.
+
+      $ sudo apt install ruby 
+    
+  c) Install VirtualBox.
+
+      $ sudo apt install virtualbox
```
Comment (imesh) [inline bosh-release/README.md]: I think it's better to consider OSX and other Linux platforms when providing instructions. May be we can ask the user to install BOSH CLI and VirtualBox. If there are recursive dependencies those could be installed by referring relevant product installation guides. WDYT?

File: bosh-release/README.md [hunk 2]
```diff
@@ -46,7 +57,7 @@ A BOSH release for deploying WSO2 API Manager 2.1.0 on BOSH Director:
 
 6. Download Oracle JDK 1.8 from Oracle website and WSO2 API Manager 2.1.0 via WSO2 Update Manager (WUM).
 
-7. Add above distributions as blobs:
+7. Add above distributions as blobs. Go inside `wso2-apim-bosh-release` and run below commands.
```
Comment (imesh) [inline bosh-release/README.md]: Would it be better to mention this as a command? Then the user can directly copy and execute it. WDYT?

Comment (bhathiya) [reply bosh-release/README.md]: +1, I'll add.

File: bosh-release/README.md (hunk 1 above)
Comment (bhathiya) [reply bosh-release/README.md]: +1

File: bosh-release/README.md [hunk 3]
```diff
@@ -1,108 +1,253 @@
...
+            -v internal_cidr=192.168.50.0/24 \
+            -v outbound_network_name=NatNetwork
 
-10. Download latest bosh-lite warden stemcell from bosh.io and upload it to BOSH Director:
-    
-    ```bash
-    wget https://s3.amazonaws.com/bosh-core-stemcells/warden/bosh-stemcell-3445.7-warden-boshlite-ubuntu-trusty-go_agent.tgz
-    bosh -e vbox upload-stemcell bosh-stemcell-3445.7-warden-boshlite-ubuntu-trusty-go_agent.tgz
-    ```
+8. Once VM with BOSH Director is running, point Bosh CLI to it, saving the environment with the alias `vbox`.
```
Comment (imesh) [inline bosh-release/README.md:188]: I think better to write the word bosh either in uppercase or lowercase; BOSH

**Comment by imesh** [comment]
Thanks for your great contribution Bhathiya! This PR is now merged to the master branch.

File: bosh-release/README.md (hunk 3 above)
Comment (bhathiya) [reply bosh-release/README.md:188]: Fixed in https://github.com/wso2/pivotal-cf-apim/pull/2

**Pull request merged**